datafusion/sql/src/expr/mod.rs

```diff
@@ -895,111 +844,75 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         let args = vec![fullstr, substr];
         Ok(Expr::ScalarFunction(ScalarFunction::new_udf(fun, args)))
     }
-    fn sql_agg_with_filter_to_expr(
```

Comment:
there is no special sqlparser struct for this kind of aggregate function anymore -- it is all unified